api/v1/routes/testimonials.py

```diff
@@ -0,0 +1,72 @@
+from fastapi import Depends, Query, APIRouter, status, HTTPException
```

Comment:
Why are you handling database queries and operations in your routes
This should be done in services